mesh/ncmesh.cpp

```diff
@@ -4285,6 +4285,19 @@ void NCMesh::GetPointMatrix(Geometry::Type geom, const char* ref_path,
             pm = PointMatrix(mid12, mid20, mid01);
          }
       }
+      else if (geom == Geometry::SEGMENT)
```

Comment:
This was also an orthogonal bugfix, this clause is needed for 1D non-conforming refinement.